src/Components/ODComponentTypes.php

```diff
@@ -16,6 +16,7 @@ class ODComponentTypes
     const SENSOR_COMPONENT_TYPE = 'sensor';
     const PLATFORM_COMPONENT_TYPE = 'platform';
     const ATTRIBUTE_FILTER_TYPE = 'attribute_filter';
+    const LANGUAGE_SERVICE_COMPONENT_TYPE = 'language_service';
```

Comment:
Let's get this updated to:

```suggestion
    const LANGUAGE_PROCESSOR_COMPONENT_TYPE = 'language_processor';
```